.github/workflows/migrations.yaml

```
@@ -0,0 +1,165 @@
+          files: ./*/migrations/**
+
+      # Summary for debugging
+      - name: Summarize
+        run: |
+          echo "services: $services" >> $GITHUB_STEP_SUMMARY
+        env:
+          services: ${{ steps.changes.outputs.all_changed_files }}
+
+  disallow-modifications:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Detect modifications in migrations
+        id: changes
+        uses: tj-actions/changed-files@v37
+        with:
+          json: "true"
+          path: "services"
+          files: ./*/migrations/**
+      - name: Don't modify existing migrations!
+        if: steps.changes.outputs.modified_files != '[]'
+        run: |
+          echo "Don't modify existing migrations! Doing so may result in unwanted behavior."
+          echo "You can ignore this message if you believe the changes made will not change the migrations,"
+          echo "for example by only adding comments. Reviewers may proceed only with caution."
+          exit 1
+
+  migrations:
+    runs-on: ubuntu-latest
+    needs: detect-changes
+    if: needs.detect-changes.outputs.services != '[]'
+    strategy:
+      matrix:
+        service: ${{ fromJson(needs.detect-changes.outputs.services) }}
+
+    env:
+      svc: ${{ matrix.service }}
+      POSTGRES_HOST: localhost
+      POSTGRES_PORT: 5432
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: postgres
+      POSTGRES_DB: postgres # will be overwritten in "Pull staging data"
+
+    services:
+      postgres:
+        image: postgres
+        # Set health checks to wait until postgres has started
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        env:
+          POSTGRES_HOST: localhost
+          POSTGRES_PORT: 5432
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: postgres
+          POSTGRES_DB: postgres
+        ports:
+          - 5432:5432
+    steps:
+      - name: Setup flyctl
+        uses: superfly/flyctl-actions/setup-flyctl@master
+        with:
+          version: 0.1.66
+      # currently ubuntu-latest ships with psql 14.x,
+      # whose pg_dump is incompatible with our postgres server (15.x)
+      - name: Setup postgres client
+        run: |
+          sudo sh -c 'echo "deb http://apt.postgresql.org/pub/repos/apt $(lsb_release -cs)-pgdg main" > /etc/apt/sources.list.d/pgdg.list'
+          wget -qO- https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo tee /etc/apt/trusted.gpg.d/pgdg.asc &>/dev/null
+          sudo apt update
+          sudo apt remove postgresql-client
+          sudo apt install postgresql-client-15 -y
+          /usr/lib/postgresql/15/bin/pg_dump --version
+      - name: Clone Repo
+        uses: actions/checkout@v3
+      - name: Open tunnel to staging db
+        run: |
+          flyctl proxy 5431:5432 -a helpwave-staging-postgres &
```

```suggestion
          flyctl proxy 5431:5432 -a helpwave-staging-postgres &
```
Make `helpwave-staging-postgres` a variable inside this workflow